web/locales/fi.json

```diff
@@ -0,0 +1,1786 @@
+        "now": "Now"
+    },
+    "left-panel": {
+        "zone-list-header-title": "Ilmastovaikutukset alueittain",
+        "zone-list-header-subtitle": "Sijoitus kulutetun sähkön hiili-intensiteetin mukaan (gCO₂eq/kWh)",
+        "search": "Etsi alueita"
+    },
+    "country-history": {
+        "carbonintensity24h": "Hiili-intensiteetti viimeisen 24 tunnin aikana",
+        "emissions24h": "Päästöt viimeisen 24 tunnin aikana",
+        "emissionsorigin24h": "Päästöjen alkuperä viimiesen 24 tunnin aikana",
+        "emissionsproduction24h": "Päästöt tuotettu viimiesen 24 tunnin aikana",
+        "electricityorigin24h": "Sähkön alkuperä viimeisen 24 tunnin aikana",
+        "electricityproduction24h": "Sähkön tuotanto viimeisen 24 tunnin aikana",
+        "electricityprices24h": "Sähkön hinnat viimeisen 24 tunnin aikana",
+        "Getdata": "Saa historiallista tietoa, marginaalinen ja ennuste API"
+    },
+    "footer": {
+        "likethisvisu": "Pidätkö visualisoinnista?",
+        "loveyourfeedback": "Lähetä palautetta!",
+        "foundbugs": "Löysitkö vikoja tai onko ideoita? Kerro niistä",
+        "faq-text": "Jotain epäselvää? Lue",
+        "faq": "usein kysytyt kysymykset.",
+        "here": "täällä"
+    },
+    "legends": {
+        "windpotential": "Tuulivoimapotentiaali",
+        "solarpotential": "Aurinkovoimapotentiaali",
+        "colorblindmode": "värisokeamoodi",
+        "carbonintensity": "Hiili-intensiteetti"
+    },
+    "tooltips": {
+        "carbonintensity": "Hiili-intensiteetti",
+        "lowcarbon": "Vähähiilinen",
+        "renewable": "Uusiutuva",
+        "crossborderexport": "Rajat ylittävä vienti",
+        "carbonintensityexport": "Viennin hiili-intensiteetti",
+        "ofinstalled": "asennetusta kapasiteetista",
+        "utilizing": "hyödyntäen",
+        "withcarbonintensity": "hiili-intensiteetin kanssa",
+        "showWindLayer": "Näytä tuulikerros",
+        "hideWindLayer": "Piilota tuulikerros",
+        "showSolarLayer": "Näytä aurinkokerros",
+        "hideSolarLayer": "Piilota aurinkokerros",
+        "toggleDarkMode": "Vaihda tummaa moodia",
+        "noParserInfo": "Tietoja ei ole saatavana",
+        "temporaryDataOutage": "Live-tiedot eivät ole väliaikaisesti saatavilla",
+        "production": "tuotanto",
+        "consumption": "kulutus",
+        "cpinfo": "<b>kuluts</b> ottaa huomioon tuonnin ja viennin, <b>tuotanto</b> jättää ne huomioimatta",
+        "selectLanguage": "Valitse kieli",
+        "lowCarbDescription": "Sisältää uusiutuvat energialähteet ja ydinvoiman"
+    },
+    "misc": {
+        "maintitle": "Sähkönkulutuksen reaaliaikiaset hiilidioksidipäästöt",
+        "oops": "Meillä on vaikeuksia palvelimen saavuttamisessa. Yritämme uudelleen muutamassa sekunnissa.",
+        "newversion": "Uusi versio saatavilla! Lataa <a onClick=\"location.reload(true);\">tästä</a>.",
```

Comment:
`Lataa` is more like "Download". I think `Päivitä` would be better, which is kinda like "Refresh"